app/templates/index.html

```diff
@@ -41,5 +41,33 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
+    <h1><u>About Us</u></h1>
```

Comment:
I propose changing this h1 to another type of header like h2. I believe it is best practice to have only 1 h1 per page, both for SEO and for screen reader accessibility (screen reader users jump between headings)

Reply:
I considered that, but I'm worried that will require changes to all the headings to make the headings make visual sense. The names are currently in header 2, which seems a little small for a section title. What do you think @MAMV3x3?

Reply:
You can adjust the font sizes in the CSS for the other titles, so don't worry about it, as @gibbonhug proposed, it's usually a best practice to only use one h1 per page

Reply:
Alright, fixed it 😃